guide/moderator/lock.md

```diff
@@ -0,0 +1,11 @@
+## User Handbook
+
+## Locking a Discussion
+
+To lock a discussion, as always you'll first need to find the discussion in threaded view, once you have go ahead and bring up the thread menu by hovering the mouse over the right of the thread, and selecting the three dots menu, as show in the screenshot below:
```

Comment:
"threaded view"

This is the (All) Discussions List. Threaded View, in forum lingo, often means something else relating to type of post display mode (where posts nest under the post they reply to, rather than the linear form we use).